app/services/jobseekers/account_transfer.rb

```diff
@@ -0,0 +1,55 @@
+class Jobseekers::AccountTransfer
+  class AccountNotFoundError < StandardError; end
+  attr_reader :current_jobseeker, :account_to_transfer
+
+  def initialize(current_jobseeker, email)
+    @current_jobseeker = current_jobseeker
+    @account_to_transfer = Jobseeker.find_by(email: email)
+  end
+
+  def call
+    raise AccountNotFoundError, "Account with email not found" unless account_to_transfer
+
+    ActiveRecord::Base.transaction do
+      transfer_profile
+      transfer_feedbacks
+      transfer_job_applications
+      transfer_saved_jobs
+      update_subscriptions
+    end
```

Comment:
Regarding the PR description comment:
> I have not implemented the hard delete of the old account. Is this necessary? It seems like it increases risk without a large benefit.

I think not deleting the old account is problematic:
- This service removes the profile, feedbacks, job_applications, saved_jobs and subscriptions from the old account. All that is left is an empty shell, with no value to the service or the user.
- The old accounts would count in the total of "active" accounts in the service, misrepresenting the number of real users.
- The old accounts would also receive communications if we send them to all active users, campaigns...
- The user may get confused and use their old credentials to find themselves with an empty account, no profile... and maybe even start executing actions from this account.

What are the possible risks on hard deleting the account once we have removed all their info?